source/common/http/conn_manager_impl.cc

```diff
@@ -1392,7 +1431,6 @@ void ConnectionManagerImpl::ActiveStream::encodeHeaders(ResponseHeaderMap& heade
     // prevent any new streams.
     connection_manager_.startDrainSequence();
     connection_manager_.stats_.named_.downstream_cx_drain_close_.inc();
-    ENVOY_STREAM_LOG(debug, "drain closing connection", *this);
```

Comment:
Why was this removed but the stat wasn't? related to my comment above about possible double stat increments.